index.html

```diff
@@ -1,3 +1,3 @@
 <html>
-	Welcome!
+	Welcome User! This is an added text for next commit.
```

Comment:
Welcome User should be Welcome Users - and can we make it dynamic and get user details on the session to display user first and last name?